Gemfile

```diff
@@ -17,6 +17,7 @@ gem 'rtoolsHCK', git: 'https://github.com/HCK-CI/rtoolsHCK.git', tag: 'v0.5.1'
 gem 'rubyzip'
 gem 'sentry-ruby'
 gem 'sorbet-runtime'
+gem 'aliyun-sdk'
```

Comment:
As you add a new gem and we use Sorbet for type checking, please regenerate Sorbet RBIs and commit with Gemfile update. `bundle exec tapioca gem` - to regenerate